pkg/reconciler/ingress/config/istio_test.go

```diff
@@ -233,14 +234,14 @@ func TestGatewayConfiguration(t *testing.T) {
 				Name:      IstioConfigName,
 			},
 			Data: map[string]string{
-				"external-gateways": `
-                  - namespace: "namespace1"
-                    name: "gateway1"
-                    service:  "istio-ingressbackroad.istio-system.svc.cluster.local"`,
-				"local-gateways": `
-                  - namespace: "namespace2"
-                    name: "gateway2"
-                    service:  "istio-local-gateway.istio-system.svc.cluster.local"`,
+				"external-gateways": replaceTabs(`
```

Comment:
Should not we have a test for defaults to:

```
{
		name: "new format alone defaults",
		wantIstio: &Istio{
			IngressGateways: []Gateway{{
				Namespace:  system.Namespace(),
				Name:       "knative-ingress-gateway",
				ServiceURL: "istio-ingressgateway.istio-system.svc.cluster.local",
			}},
			LocalGateways: []Gateway{{
				Namespace:  system.Namespace(),
				Name:       "knative-local-gateway",
				ServiceURL: "knative-local-gateway.istio-system.svc.cluster.local",
			}},
		},
		config: &corev1.ConfigMap{
			ObjectMeta: metav1.ObjectMeta{
				Namespace: system.Namespace(),
				Name:      IstioConfigName,
			},
		},
	}
```

Reply:
There is the very first one

```golang
	{
		name: "gateway configuration with no network input",
		wantIstio: &Istio{
			IngressGateways: defaultIngressGateways(),
			LocalGateways:   defaultLocalGateways(),
		},
		config: &corev1.ConfigMap{
			ObjectMeta: metav1.ObjectMeta{
				Namespace: system.Namespace(),
				Name:      IstioConfigName,
			},
		},
	}, 
```

Reply:
(default doesn't matter if it's the old format or the new one)

Reply:
ok that's good then.